.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+  "presets": ["es2015"],
+  "env": {
+    "test": {
+      "plugins": [ "istanbul" ]
```

Comment:
Yes, [NYC](https://github.com/istanbuljs/nyc#use-with-babel-plugin-istanbul-for-es6es7es2015-support) uses it